src/paymentRequest/IncomingPaymentRequestListItem.test.tsx

```diff
@@ -326,4 +332,54 @@ describe('IncomingPaymentRequestListItem', () => {
 
     expect(navigate).toBeCalledTimes(0)
   })
+
+  it('emits correct analytics event when CTA button is pressed', () => {
+    const store = createMockStore({
+      identity: identityLoading,
+      tokens: balances,
+    })
+
+    const { getByTestId } = render(
+      <Provider store={store}>
+        <IncomingPaymentRequestListItem {...props} />
+      </Provider>
+    )
+
+    const remindButton = getByTestId(
+      'IncomingPaymentRequestNotification/FAKE_ID_2/CallToActions/send/Button'
+    )
+    fireEvent.press(remindButton)
```

Comment:
i generally prefer using `getByText` rather than `getByTestId` wherever possible because it tests closer to what a user would see. this [article](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text) resonated with me, but of course it is just an opinion and we haven't been very consistent about this perspective in our projects